ibm/mas_devops/common_tasks/get_ingress_cert.yml

```diff
@@ -17,7 +17,7 @@
   set_fact:
     found_router_default_secret: true
     cluster_ingress_secret_name: router-certs-default
-    cluster_ingress_tls_crt: "{{ router_certs_default_secret.data['tls.crt'] | b64decode }}"
+    cluster_ingress_tls_crt: "{{ router_certs_default_secret.resources[0].data['tls.crt'] | b64decode }}"
```

Comment:
@ctrafa  I see @durera added this file in a PR related to ROSA recently... question: did you get that bug by testing in ROSA [here](https://github.com/ibm-mas/ansible-devops/pull/450/files#diff-e6999ab814ed63bdb1864b1f90d7e2e37f27ff862102b83bf158fdabbbb5d734R20)... are you performing tests also in ROSA and got this bug from it? I am concerning if the location of the ingress certificate lives in different places depending on the cluster/cloud we are testing.

Reply:
This has happened to me when deploying against Fyre.